polling_stations/apps/file_uploads/utils.py

```diff
@@ -12,17 +13,43 @@ def get_domain(request):
     return os.environ.get("APP_DOMAIN", request.META.get("HTTP_HOST"))
 
 
+# This is a quick fix: Peter can raise a
+# PR to update CUSTOM_DOMAINS with the the actual
+# list of email domains and councils as the
+# need arises.
+def approved_custom_domain(user, email_domain):
```

Comment:
With my other comment...

Can you change this to an action like "assign_approved_custom_domains" so we know it does something rather than just checks something?

And with the early return, we don't need the `else` statement, we can just dedent the rest of the code.

Reply:
Fixed with 9c15a6cd5cf7a4948cfa5c3c331209fe1a0017c2